.gitignore

```diff
@@ -106,3 +106,8 @@ src/sous_chef/static
 
 # Ignore generated pdf files.
 src/*.pdf
+
+# the magic docker-compose override file is for overriding settings locally
+# it should not be a part of the repo. If you want per-environment settings,
+# use an explicit `docker-compose -f docker-compose.yml -f docker-compose.environment.yml`
+docker-compose.override.yml
```

Comment:
@kousu  I agree with you personally. I don't like the use of the word "override" of docker-compose either and it brought me some troubles... but I think this is their convention according to their doc, so I would follow the usage of "override" so that we work on the same page with docker-experienced developers. 

I think it is important to mention the doc link somewhere in the project, in order to help developers like me and @kousu who don't know this convention of docker-compose a priori.